src/types.ts

```diff
@@ -20,7 +20,7 @@ export function declareType(type: IType) {
     return Reflect.metadata(typeMetadataKey, type);
 }
 
-export const primitiveTypeToValue: Record<number, string> = {
+const primitiveTypeToValue: Record<number, string> = {
```

Comment:
This is internal part of Node.js SDK that can be changed (see uuid type fix), so I prefer to avoid the export.
The good alternative is TypedValues usage that was described in the updated commit message.